components/framework-axp/src/main/java/framework/logging/LazyLogger.java

```diff
@@ -0,0 +1,110 @@
+package framework.logging;
+
+import java.util.function.Supplier;
+
+/**
+ * Facade to support application logging.
+ * following log level operation support is available through this class.
+ * <p>
+ * * TRACE
+ * * INFO
+ * * DEBUG
+ * * WARN
+ * * ERROR
+ * * FATAL
+ * <p>
+ * Create an instance using {@link LazyLoggerFactory#getLogger(Class)}
+ */
+public interface LazyLogger
```

Comment:
Please break this into three PRs. 
LazyLogger in one.
AXPCache into a second one.
Configuration Service into another.
Since the changes with loggers are small. We can merge it quickly.